### DISCUSSION TITLE
New Scrapers

### DISCUSSION THREAD

**Discussion by bunfire** [adultmm/AdultMediaManager#9] [Q&A]
Fantastic work.. amazing. I've been curating my personal collection for the last couple of days and it's going perfect and my plex has never looked so fucking good.

Is adding a new scraper easy enough? any chance to add http://max-hardcore.com

**Comment by adultmm** [comment]
Hi Bunfire,

I'm glad you find my job useful. 

As I can see, it is possible to create max-hardcore.com scraper. I've added it to my todo list. In the future, it is better to [create an issue ](https://github.com/adultmm/AdultMediaManager/issues) to request a scraper.

**Reply by bunfire** [reply]
#7  Thanks!
